.github/workflows/integration_tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: "**/node_modules"
-          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}-v1
```

Comment:
Pretty much the same comment as the build.yml one, could integration_tests.yml be renamed to `integration_tests_v1.yml` with no conflicts?

Reply:
done!